cypress/e2e/regression/create_safe_simple_2.cy.js

```diff
@@ -0,0 +1,125 @@
+import * as constants from '../../support/constants'
+import * as main from '../pages/main.page'
+import * as createwallet from '../pages/create_wallet.pages'
+import * as owner from '../pages/owners.pages'
+import * as ls from '../../support/localstorage_data.js'
+import * as safe from '../pages/load_safe.pages'
+
+const ownerSepolia = ['Automation owner Sepolia']
+const ownerName = 'Owner name'
+const owner1 = 'Owner1'
+
+describe('Safe creation tests 2', () => {
+  beforeEach(() => {
+    cy.visit(constants.welcomeUrl + '?chain=sep')
+    cy.clearLocalStorage()
+    main.acceptCookies()
+  })
+
+  it('Cancel button cancels safe creation', () => {
+    owner.waitForConnectionStatus()
+    createwallet.clickOnCreateNewSafeBtn()
+    safe.clickOnNextBtn()
+    createwallet.clickOnBackBtn()
+    createwallet.cancelWalletCreation()
+  })
+
+  // Owners and confirmation step
+  it('Verify Next button is disabled when address is empty', () => {
+    owner.waitForConnectionStatus()
+    createwallet.clickOnCreateNewSafeBtn()
+    safe.clickOnNextBtn()
+    safe.clearOwnerAddress(0)
+    createwallet.verifyNextBtnIsDisabled()
+  })
+
+  it('Verify owner names are autocompleted if they are present in the Address book ', () => {
+    owner.waitForConnectionStatus()
+    main
+      .addToLocalStorage(constants.localStorageKeys.SAFE_v2__addressBook, ls.addressBookData.sameOwnerName)
+      .then(() => {
+        cy.visit(constants.welcomeUrl + '?chain=sep')
+        createwallet.clickOnCreateNewSafeBtn()
+        safe.clickOnNextBtn()
+        safe.verifyOwnerNamesInConfirmationStep(ownerSepolia)
+      })
+  })
+
+  it("Verify names don't autofill if they are added to another chain's Address book", () => {
+    owner.waitForConnectionStatus()
+    main
+      .addToLocalStorage(constants.localStorageKeys.SAFE_v2__addressBook, ls.addressBookData.sameOwnerName[1])
+      .then(() => {
+        cy.visit(constants.welcomeUrl + '?chain=sep')
+        createwallet.clickOnCreateNewSafeBtn()
+        safe.clickOnNextBtn()
+        safe.verifyDataDoesNotExist(ownerSepolia)
+      })
+  })
+
+  it('Verify an valid name for owner can be inputed', () => {
+    owner.waitForConnectionStatus()
+    createwallet.clickOnCreateNewSafeBtn()
+    safe.clickOnNextBtn()
+    safe.inputOwnerName(0, ownerName)
+    safe.verifyownerNameFormatIsValid()
+  })
+
+  it('Verify Empty address field not are not allowed', () => {
```

Comment:
Looks one to one with it'Verify Next button is disabled when address is empty', - suggest to leave one of them

Reply:
Will address in next PR

Reply:
Done